apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -255,6 +254,77 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
     expect(email.channel).to.equal(ChannelTypeEnum.EMAIL);
   });
 
+  it('should correctly set expiration date (TTL) for notification and messages', async function () {
+    const { data: body } = await axiosInstance.post(
+      `${session.serverUrl}${eventTriggerPath}`,
+      {
+        name: template.triggers[0].identifier,
+        to: {
+          subscriberId: subscriber.subscriberId,
+        },
+        payload: {
+          firstName: 'Testing of User Name',
+          urlVar: '/test/url/path',
+        },
+      },
+      {
+        headers: {
+          authorization: `ApiKey ${session.apiKey}`,
+        },
+      }
+    );
+
+    await session.awaitRunningJobs(template._id);
+
+    const notifications = await notificationRepository.findBySubscriberId(session.environment._id, subscriber._id);
+
+    expect(notifications.length).to.equal(1);
+
+    const notification = notifications[0];
+
+    let expireAt = new Date(notification?.expireAt as string);
+    let createdAt = new Date(notification?.createdAt as string);
+
+    let subExpireMonths = subMonths(expireAt, 1);
+    let diff = differenceInMilliseconds(subExpireMonths, createdAt);
+
+    expect(diff).to.approximately(0, 1);
+
+    const messages = await messageRepository.findBySubscriberChannel(
+      session.environment._id,
+      subscriber._id,
+      ChannelTypeEnum.IN_APP
+    );
+
+    expect(messages.length).to.equal(1);
+    const message = messages[0];
+
+    expireAt = new Date(message?.expireAt as string);
+    createdAt = new Date(message?.createdAt as string);
+
+    subExpireMonths = subMonths(expireAt, 6);
+    diff = differenceInMilliseconds(subExpireMonths, createdAt);
+
+    expect(diff).to.approximately(0, 1);
+
+    const emails = await messageRepository.findBySubscriberChannel(
+      session.environment._id,
+      subscriber._id,
+      ChannelTypeEnum.EMAIL
+    );
+
+    expect(emails.length).to.equal(1);
+    const email = emails[0];
+
+    expireAt = new Date(email?.expireAt as string);
+    createdAt = new Date(email?.createdAt as string);
+
+    subExpireMonths = subMonths(expireAt, 1);
+    diff = differenceInMilliseconds(subExpireMonths, createdAt);
+
+    expect(diff).to.approximately(0, 1);
```

Comment:
Is it possible to create constants somehwere in the top of the case so we can easily understand the intended behaviour? For example:
```
const IN_APP_MESSAGE_EXPIRE_MONTHS = 6;
const MESSAGE_EXPIRE_MONTHS = 1;
```

Right now need to manually do the logic to understand that the second param of the subMonths function is for this value.

Some more read about it: https://www.pluralsight.com/tech-blog/avoiding-magic-numbers/ :)